### PR TITLE
Use Docker at Travis

### PR DESCRIPTION
**Pull request by imobachgs**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+language: ruby
+services:
+  - docker
+
+before_install:
+  - docker build -t yast-theme-image .
+script:
+  # the "yast-travis-ruby" script is included in the base yastdevel/ruby image
+  # see https://github.com/yast/docker-yast-ruby/blob/master/yast-travis-ruby
+  - docker run -it -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-theme-image yast-travis-ruby
```
Comment (teclator) [inline .travis.yml]: yast-theme-image?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  yast2-mail \
+  yast2-network \
+  yast2-nfs-client \
+  yast2-nfs-server \
+  yast2-nis-client \
+  yast2-nis-server \
+  yast2-ntp-client \
+  yast2-online-update-configuration \
+  yast2-printer \
+  yast2-proxy
```
Comment (teclator) [inline Dockerfile]: `&& zypper clean -a`?

Comment (imobachgs) [inline Dockerfile]: Good point. We would remove some metadata, as packages are not cached anyway IIRC.

**Comment by imobachgs** [comment]
Thanks!

**Pull request merged**